.gitignore

```diff
@@ -3,4 +3,6 @@
 /npm-debug.log
```

Comment:
There is a super cool intellij plugin called .gitignore. It generates the a complete git ignore which adds the ignores for:
* Node
* Intellij
etc.

Maybe generate one with this.

Reply:
I created a task for this.